specification/dimensions/usagetype.md

```diff
@@ -0,0 +1,42 @@
+# Usage Type
+
+UsageType is a categorization parameter that defines how a resource or service was consumed during a specified period. The UsageType can influence the price point and also indicate whether purchased commitments are used or unused. This classification offers a nuanced view into the resource or service's consumption patterns and helps in understanding how you're billed, especially when considered alongside attributes like ChargeType and PurchaseType.
+
+The UsageType column MUST be present and MUST NOT be null or empty. This column is of type String and MUST be one of the
+allowed values.
+
+See [Appendix: Usage Type](#usagetype-1) for details about allowed values and governance criteria for this dimension.
+
+## Column ID
+
+UsageType
+
+## Display Name
+
+Usage Type
+
+## Description
+
+Indicates whether the record pertains to an on-demand or recurring charge, or relates to the cost or capacity using a committed discount rate or from unutilized commitments.
+
+## Content Constraints
+
+| Constraint      | Value                                    |
+| :-------------- | :--------------------------------------- |
+| Column required | True                                     |
+| Data type       | String                                   |
+| Allows nulls    | False                                    |
+| Value format    | list-of-values                           |
+
+Allowed values:
+
+| Value      | Description                                                                                                                                                                   |
+|:-----------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| On-Demand         | This usage type describes the consumption of resources or services that are billed at the standard on-demand unit price.                                                |
```

Comment:
Maybe OnDemand (without -) is simpler?

How would we deal with Google Sustained Usage Discount?
Would it be another value for UsageType, or a particular case for UsageType=OnDemand. In the latter case, which dimension would bear this information?

Reply:
+1 on SUDs being volume pricing and not a discount.

Reply:
I'm not so sure about that because it's not really volume. You can consume the same "744 core-hours" of compute by maximizing SUD or not depending on how your organize your workloads.

And Practionners work on how to optimize their SUD  in a similar way as they work on optimizing their commitment discounts :
 - Avoid spikes in instance numbers (if you spread your compute usage, you maximize your SUD)
 - Use consistent instance families (if you use N2 all around instead of "N1 here and N2 there", you maximize your SUD)
 
So i understand the point about considering it a "volume pricing" but there are also aspects which favor considering it as a discount like CUD or RIs.

Reply:
@nfonrose, what value would you want to see for that? It's not a commitment. 

/cc @rodneyjoyce for `PricingModel` 🙂

Reply:
Indeed, SUD could be managed with another dimension, but for simplicity of the FOCUS model, this information could be put in the same dimension. The value could be simply 'Sustained'.

Reply:
Keeping this comment active until the PricingModel PR comes out. Will update everyone here when that's ready.

Reply:
Can everyone please go review the PricingModel PR and leave comments there for anything in this area?
See #214